oppijanumerorekisteri-service/src/main/java/fi/vm/sade/oppijanumerorekisteri/controllers/Service2ServiceController.java

```diff
@@ -58,7 +58,7 @@ public String oidByHetu(@PathVariable String hetu) {
     @PreAuthorize("hasRole('APP_HENKILONHALLINTA_OPHREKISTERI')")
     @RequestMapping(value = "/duplicateHenkilos", method = RequestMethod.POST) 
     public List<HenkiloViiteDto> findDuplicateHenkilos(@RequestBody HenkiloCriteria criteria) {
-        return this.henkiloService.findHenkiloViittees(criteria);
+        return this.henkiloService.findHenkiloViittees(criteria, 5000);
```

Comment:
Ja tarkemmin ajatellen, kannattaisiko splitSize jättää `HenkiloServiceImpl` :in sisälle? Sehän on kutsujalle kuitenkin vain implementation detail, joka voisi olla siistimpää piilottaa sinne.